src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -167,9 +170,9 @@ module.exports = class LevelPlane {
       this.determineRailType(position, true);
 
       if (this.levelModel && this.levelModel.controller.levelView) {
-        const northEast = [position[0] + 1, position[1] - 1];
-        const southWest = [position[0] - 1, position[1] + 1];
-        let positionAndTouching = this.getOrthogonalPositions(position).concat([position, northEast, southWest]);
+        const northEast = Position.add(position, [1, -1]);
+        const southWest = Position.add(position, [-1, 1]);
```

Comment:
Worth adding `position.south().west()`?

Until everything is a `Position` instance it would be `Position.west(Position.south(position))` 🙁.